metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -97,6 +97,7 @@
 public class BigtableBackend extends AbstractMetricBackend implements LifeCycles {
     private static final Logger log = LoggerFactory.getLogger(BigtableBackend.class);
 
+    public static final int MAX_KEY_ROW_SIZE = 4000;
```

Comment:
maybe add a comment here like the one below has for more context

Reply:
done